src/excmds.ts

```diff
@@ -1221,7 +1221,10 @@ export async function scrollpx(a: number, b: number) {
     Note that if `a` is 0 or 100 and if the document is not scrollable in the given direction, Tridactyl will attempt to scroll the first scrollable element until it reaches the very bottom of that element.
 */
 //#content
-export function scrollto(a: number, b: number | "x" | "y" = "y") {
+export function scrollto(a: number | string, b: number | "x" | "y" = "y") {
```

Comment:
b should probably accept any arbitrary vector to scroll along, e.g, "x+y", with perhaps allowing "z" for zoom.

I think we should probably turn it into a unit vector - other behaviour might be surprising and will avoid requiring users from having to type in irrational numbers.

Reply:
How thoughtful. :+1: on the unit vector.